cfme/tests/cli/test_appliance_console.py

```diff
@@ -22,6 +22,8 @@
 from cfme.utils.version import LOWEST
 
 
+SECONDARY_DNS_REGEX = r'(Secondary DNS:\s*)(?P<secondary_dns>\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})'
```

Comment:
I don't think this covers all possible DNS possibilities but I see it is used `test_appliance_console`.

Reply:
yes, we are just fetching ip and validating ip here